Assets/MRTK/SDK/Features/UX/Prefabs/Panels/ToggleFeaturesPanel.prefab

```diff
@@ -125,21 +126,51 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: cf12ee76e7e00a44a9a84256760020e6, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  nodeList: []
+  nodeList:
+  - Name: ToggleProfilerButton
```

Comment:
Small question on the toggle features panel: does it make sense for the one we use in our example scene to be the SDK prefab? It feels very specific to our scenario. It might be interesting to have a generic one, with ours being an example prefab variant or something.